src/misc/ssEconomy.lua

```diff
@@ -20,6 +21,10 @@ function ssEconomy:load(savegame, key)
     self.aiDayEnd = ssXMLUtil.getFloat(savegame, key .. ".settings.aiDayEnd", 18)
     self.loanMax = ssXMLUtil.getFloat(savegame, key .. ".settings.loanMax", 1500000)
     self.baseLoanInterest = ssXMLUtil.getFloat(savegame, key .. ".settings.baseLoanInterest", 10)
+
+    -- Factors used to convert vanilla AI prices to Seasons AI prices
+    self.aiPriceFactor = self.aiPricePerHourWork / ssEconomy.VANILLA_AI_PER_HOUR
```

Comment:
Found a bug: :load is not called on a multiplayer client. So you should move these 3 lines to :loadMap instead